Vulkan-Pipeline-Assistant/vulkanrenderer.cpp

```diff
@@ -46,6 +46,7 @@ void VulkanRenderer::releaseResources() {
     delete m_shaderAnalytics;
     if (m_vertexInput) delete m_vertexInput;
     if (m_descriptors) delete m_descriptors;
+    //TODO Delete config viewports
```

Comment:
Implement deletion to avoid putting a memory leak in master and remove todo

Reply:
Done in latest push